src/ProjectSystemTools/TableControl/TableKeyNames.cs

```diff
@@ -11,7 +11,8 @@ internal static class TableKeyNames
         public const string Elapsed = "elapsed";
         public const string Status = "status";
         public const string Targets = "targets";
-        public const string LogPath = "logpath";
+        public const string BuildID = "buildid";
```

Comment:
What's the rationale for adding a column for this?

Reply:
I needed some way to get the build ids for builds selected on the UI, so that I can get the logs back from the server side using their ids
Although if this change modifies the UI, it's probably not a good idea since the user (probably) shouldn't have to think about build Ids

Reply:
I see. Adding this const won't make it show in the UI. This is fine as is. I took a quick look but didn't find a way to access the `UIBuildSummary` directly from the `ITableEntry`, so this approach makes sense to me now.